search/src/androidMain/kotlin/com/islamversity/search/view/SurahRowView.kt

```diff
@@ -21,16 +21,17 @@ class SurahRowView @JvmOverloads constructor(
     attributeSet: AttributeSet? = null
 ) : LinearLayout(context, attributeSet) {
 
-    private val binding = RowSuraBinding.inflate(LayoutInflater.from(context), this, true)
+    private val binding = RowSurahBinding.inflate(LayoutInflater.from(context), this, true)
 
 
     lateinit var uiItem: SurahUIModel
```

Comment:
Doesn't need to keep a reference of the model! when the `listener` method of SurahRowView is called you have access to the model.